### PR TITLE
symlink NFSv3 operation

### PR DESCRIPTION
**Pull request by ploki**

### CODE REVIEW

File: src/node_nfsc_symlink3.cc [hunk 1]
```diff
@@ -0,0 +1,164 @@
...
+    success = true;
+}
+
+void NFS::SymLink3Worker::HandleOKCallback()
+{
+    Nan::HandleScope scope;
+    if (success) {
+        v8::Local<v8::Value> obj_fh;
+        if (res.SYMLINK3res_u.resok.obj.handle_follows) {
+            obj_fh = Nan::NewBuffer(res.SYMLINK3res_u.resok.obj.post_op_fh3_u.handle.data.data_val,
```
Comment (LaurenSpiegel) [inline src/node_nfsc_symlink3.cc]: very nesty

File: tests/functional/010_nfscV3.js [hunk 2]
```diff
@@ -224,6 +224,17 @@ async.waterfall([
...
+        describeIt('should create a symbolic link', done => {
+            var name = 'bar_' + crypto.randomBytes(8).toString('hex');
+            mnt.symlink(root_fh, name, {mode: 0o777}, 'foo-bar-baz',
+                        (err, object, obj_attrs, dir_wcc) => {
+                            assert.strictEqual(err, null);
+                            assert.strictEqual(typeof(object), 'object');
+                            assert.strictEqual(obj_attrs.type, mnt.NF3LNK);
+                            done(next, null);
+                        });
+        }),
```
Comment (LaurenSpiegel) [inline tests/functional/010_nfscV3.js]: one test?  aren't there any edge/corner cases?

File: src/node_nfsc_symlink3.cc (hunk 1 above)
Comment (ploki) [inline src/node_nfsc_symlink3.cc]: yes! thanks to the RPC autogenerated code from rpc/nfs3.x :)

File: tests/functional/010_nfscV3.js (hunk 2 above)
Comment (ploki) [inline tests/functional/010_nfscV3.js]: yes, there are hundred of corner cases. tests efforts is focused on the NFS server using this module
